smarty/templates/main.tpl

```diff
@@ -124,16 +124,16 @@
                    </div>
                    <div class="collapse navbar-collapse" id="example-navbar-collapse">
                         <ul class="nav navbar-nav">
-                            {foreach from=$menus item=menuitems key=category}
+		            {section name=category loop=$menus}
                                  <li class="dropdown">
-                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">{$category}<b class="caret"></b>
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">{$menus[category].Category}<b class="caret"></b>
                                         <ul class="dropdown-menu">
-                                            {section name=itemloop loop=$menuitems}
-                                            <li><a href="{$menuitems[itemloop]->getLink()}">{$menuitems[itemloop]->getLabel()}</a></li>
+                                            {section name=item loop=$menus[category].Items}
+                                            <li><a href="{$menuitems[itemloop]->link}">{$menus[category].Items[item]->label}</a></li>
```

Comment:
{$menuitems[itemloop]->link} will return null. All the url link in the menu doesn't work.

Reply:
Oops, thanks. Fixed!